assets/robotest/run.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly TARGET=${1:?Usage: [path to config]}
+
+GRAVITY_URL=${GRAVITY_URL:?Set GRAVITY_URL to the current gravity binary}
+IMAGEDIR_MOUNTPOINT=${IMAGEDIR_MOUNTPOINT:? Set IMAGEDIR_MOUNTPOINT to the mountpoint where images will be availabe inside the container}
+INSTALLER_URL=${INSTALLER_URL:?Set INSTALLER_URL to the container local default robotest image}
+OPSCENTER_URL=${OPSCENTER_URL:?Set OPSCENTER_URL to the container local opscenter image}
+TELEKUBE_URL=${TELEKUBE_URL:?Set TELEKUBE_URL to the container local telekube image}
+IMAGEDIR=${IMAGEDIR:? Set IMAGEDIR to the directory with robotest images}
+STATEDIR=${STATEDIR:?Set STATEDIR to a suitable place to store robotest terraform state and logs}
+
+TAG=$(git rev-parse --short HEAD)
+
+# cloud provider that test clusters will be provisioned on
+# see https://github.com/gravitational/robotest/blob/v2.0.0/infra/gravity/config.go#L72
+DEPLOY_TO=${DEPLOY_TO:-gce}
+GCL_PROJECT_ID=${GCL_PROJECT_ID:-"kubeadm-167321"}
+GCE_REGION="northamerica-northeast1,us-west1,us-east1,us-east4,us-central1"
+# GCE_VM tuned down from the Robotest's 7 cpu default in 09cec0e49e9d51c3603950209cec3c26dfe0e66b
+# We should consider changing Robotest's default so that we can drop the override here. -- 2019-04 walt
+GCE_VM=${GCE_VM:-custom-4-8192}
+GCE_PREEMPTIBLE=${GCE_PREEMPTIBLE:-'false'}
+# Parallelism & retry, tuned for GCE
+PARALLEL_TESTS=${PARALLEL_TESTS:-4}
+REPEAT_TESTS=${REPEAT_TESTS:-1}
+RETRIES=${RETRIES:-3}
+FAIL_FAST=${FAIL_FAST:-false}
+ALWAYS_COLLECT_LOGS=${ALWAYS_COLLECT_LOGS:-true}
+
+# what should happen with provisioned VMs on individual test success or failure
+DESTROY_ON_SUCCESS=${DESTROY_ON_SUCCESS:-true}
+DESTROY_ON_FAILURE=${DESTROY_ON_FAILURE:-true}
+
+EXTRA_VOLUME_MOUNTS="-v $IMAGEDIR:$IMAGEDIR_MOUNTPOINT:ro"
+
+export INSTALLER_URL TELEKUBE_URL OPSCENTER_URL IMAGEDIR_MOUNTPOINT
+SUITE=$($TARGET configuration)
+
+# GRAVITY_FILE/GRAVITY_URL specify the location of the up-to-date gravity binary
+if [ -d $(dirname ${GRAVITY_URL}) ]; then
+  GRAVITY_FILE='/robotest-bin/'$(basename ${GRAVITY_URL})
+  EXTRA_VOLUME_MOUNTS=${EXTRA_VOLUME_MOUNTS:-}" -v "$(dirname ${GRAVITY_URL}):$(dirname ${GRAVITY_FILE}:ro)
+fi
+
+check_files () {
+	ABORT=
+	for v in $@ ; do
+		if [ ! -f "${v}" ] ; then
+			echo "${v} does not exist"
```

Comment:
nit: output errors to stderr? `>&2 echo ...`

Reply:
I'm conflicted on this: it is a small change, but would also cause 7.0.x to unnecessarily diverge from master, resulting in a chance of future merge conflicts.  I'll pick it up in master first, and then backport it in a later bulk move.